cosmic/src/kick.f

```diff
@@ -191,18 +201,30 @@ SUBROUTINE kick(kw,m1,m1n,m2,ecc,sep,jorb,vk,snstar,
           vk2 = v(1)*v(1) + v(2)*v(2) + v(3)*v(3)
           vk = SQRT(vk2)
 
+          if(kickflag.eq.0)then
 * Limit BH kick with fallback mass fraction.
-          if(kw.eq.14.and.bhflag.eq.0)then
-             vk2 = 0.d0
-             vk = 0.d0
-          elseif(kw.eq.14.and.bhflag.eq.1)then
-              fallback = MIN(fallback,1.d0)
-              vk = MAX((1.d0-fallback)*vk,0.d0)
-              vk2 = vk*vk
-          elseif(kw.eq.14.and.bhflag.eq.2)then
-             vk = vk * mxns / m1n
-             vk2 = vk*vk
+             if(kw.eq.14.and.bhflag.eq.0)then
+                vk2 = 0.d0
+                vk = 0.d0
+             elseif(kw.eq.14.and.bhflag.eq.1)then
+                 fallback = MIN(fallback,1.d0)
+                 vk = MAX((1.d0-fallback)*vk,0.d0)
+                 vk2 = vk*vk
+             elseif(kw.eq.14.and.bhflag.eq.2)then
+                vk = vk * mxns / m1n
+                vk2 = vk*vk
+             endif
+          elseif(kickflag.eq.-1)then
```

Comment:
Does kickflag = [-1, -2] rely on remnantflag to be >= 3  since the remnant masses derived for mean_mns depend on the Fryer 2012 models? (That's my takeaway from the Giacobbo and Mapelli paper). If this is the case, we should throw an error if remnant flag < 3.

Reply:
@katiebreivik that's not a bad idea. One we should also consider is increasing the proto-core mass to 1.2. This isn't done in Fryer 2012, but Mario said that Chris recommended doing this to match NS masses more. Thoughts?

Reply:
On second thought, I think it would be best not to raise an error. This won't really affect things, and remnant flags < 3 probably won't be used anyways. The assumption of the mean mns can be independent from the choice of kicks. 

However, I'm asking Mario about changing mean_mns from 1.0 to 1.1. See left column of https://arxiv.org/pdf/1909.06385.pdf page 3, last paragraph.